generated/index.md

```diff
@@ -11828,8 +14648,17 @@
 * [2021-03-01](web/microsoft.web/2021-03-01/types.md#resource-microsoftwebsites2021-03-01)
 
 ### microsoft.web/sites/backups
-* [2015-08-01](web/microsoft.web/2015-08-01/types.md#resource-microsoftwebsitesbackups2015-08-01)
-* [2016-08-01](web/microsoft.web/2016-08-01/types.md#resource-microsoftwebsitesbackups2016-08-01)
+* [2018-02-01](web/microsoft.web/2018-02-01/types.md#resource-microsoftwebsitesbackups2018-02-01)
+* [2018-11-01](web/microsoft.web/2018-11-01/types.md#resource-microsoftwebsitesbackups2018-11-01)
+* [2019-08-01](web/microsoft.web/2019-08-01/types.md#resource-microsoftwebsitesbackups2019-08-01)
+* [2020-06-01](web/microsoft.web/2020-06-01/types.md#resource-microsoftwebsitesbackups2020-06-01)
+* [2020-09-01](web/microsoft.web/2020-09-01/types.md#resource-microsoftwebsitesbackups2020-09-01)
+* [2020-10-01](web/microsoft.web/2020-10-01/types.md#resource-microsoftwebsitesbackups2020-10-01)
+* [2020-12-01](web/microsoft.web/2020-12-01/types.md#resource-microsoftwebsitesbackups2020-12-01)
+* [2021-01-01](web/microsoft.web/2021-01-01/types.md#resource-microsoftwebsitesbackups2021-01-01)
+* [2021-01-15](web/microsoft.web/2021-01-15/types.md#resource-microsoftwebsitesbackups2021-01-15)
+* [2021-02-01](web/microsoft.web/2021-02-01/types.md#resource-microsoftwebsitesbackups2021-02-01)
+* [2021-03-01](web/microsoft.web/2021-03-01/types.md#resource-microsoftwebsitesbackups2021-03-01)
```

Comment:
Same situation as described above for `Microsoft.Maintenance/applyUpdates` resource. 

This resource defines a [GET by variable name operation](https://github.com/Azure/azure-rest-api-specs/blob/98e17cdd0a433198aa964963f5178ce2f0d8f2b4/specification/web/resource-manager/Microsoft.Web/stable/2016-08-01/WebApps.json#L485-L568) and a [PUT by constant name operation](https://github.com/Azure/azure-rest-api-specs/blob/98e17cdd0a433198aa964963f5178ce2f0d8f2b4/specification/web/resource-manager/Microsoft.Web/stable/2016-08-01/WebApps.json#L440-L484) in the `2015-08-01` and `2016-08-01` versions. Later versions do not include the constant-name PUT operation, just the variable name GET operation.